btsieve/src/main.rs

```diff
@@ -201,8 +201,16 @@ fn create_ethereum_routes(
 
             let web3_client = web3_client.clone();
 
-            let blocks = ethereum_block_listener(web3_client.clone(), settings.poll_interval_secs)
-                .expect("Should return a Web3 block poller");
+            let blocks = ethereum_block_listener(
+                web3_client.clone(),
+                Duration::from_secs(match network {
+                    EthereumNetwork::Mainnet => 5,
```

Comment:
:question: Any reason why the Mainnet value went from 17 to 5?

Reply:
I think 5 is better :)